cmd/minishift/cmd/start.go

```diff
@@ -374,8 +374,7 @@ func determineClusterUpParameters(config *clusterup.ClusterUpConfig) map[string]
 	viper.Set(startFlags.HostDataDir.Name, viper.GetString(startFlags.HostDataDir.Name))
 	viper.Set(startFlags.HostVolumeDir.Name, viper.GetString(startFlags.HostVolumeDir.Name))
 	viper.Set(startFlags.HostPvDir.Name, viper.GetString(startFlags.HostPvDir.Name))
-
-	setDefaultRoutingPrefix(config.Ip)
+	viper.Set(startFlags.RoutingSuffix.Name, config.RoutingSuffix)
```

Comment:
Just to confirm, you are doing this to make sure if user has set any routing suffix it gets passed to startFlags right?